### ISSUE TITLE
Not compatible with jedis-3.1.0

### ISSUE THREAD

**Issue by bwrega** [kmtong/logback-redis-appender#23]
com.cwbase.logback.RedisAppender calls methods redis.clients.jedis.JedisPool.returnBrokenResource(...) and redis.clients.jedis.JedisPool.returnResource(...)
that are marked as protected in jedis-3.1.0 (and my guess is that it's disabled since jedis-3.0.0).

redis.clients.jedis.Jedis.close() should be called instead

**Comment by derkoe** [comment]
Seems like this is fixed with #22

**Comment by drapej** [comment]
Yes, i think too, but the version 1.1.6 is not released and not in the maven central repo.
